### DISCUSSION TITLE
jspsych-react

### DISCUSSION THREAD

**Discussion by subashsubbu** [jspsych/jsPsych#1957] [Q&A] [answered]
I was trying to run jspsych experiment in react for my project. Experiment is about displaying slideshow of images for now. When I run the app it gives me error in the console saying no plugin loaded and after going to second step which is displaying images it throws another error saying undefined is not an object. I think I have imported the plugin properly and I don't where I did wrong. I have posted all my screenshots.
![Screen Shot 2021-07-01 at 11 55 39 AM](https://user-images.githubusercontent.com/35239091/124161738-640b1f00-da63-11eb-9788-5b4336eb26cb.png)
![Screen Shot 2021-07-01 at 11 56 07 AM](https://user-images.githubusercontent.com/35239091/124161750-666d7900-da63-11eb-809a-42f7f32927e6.png)
![Screen Shot 2021-07-01 at 11 56 19 AM](https://user-images.githubusercontent.com/35239091/124161758-69686980-da63-11eb-84db-a9f4ce4177f5.png)
![Screen Shot 2021-07-01 at 11 57 41 AM](https://user-images.githubusercontent.com/35239091/124161948-9ae13500-da63-11eb-814d-c40306cf0abe.png)

**Comment by jodeleeuw** [comment, marked as answer]
Hi @subashsubbu ,

`jspsych-react` is not an "official" jspsych product. I'm not sure who, if anyone, is offering support for it right now.

**Reply by subashsubbu** [reply]
I see. Thank you @jodeleeuw
